### ISSUE TITLE
can not run config

### ISSUE THREAD

**Issue by HugoPhi** [lukas-reineke/indent-blankline.nvim#941]
### Problem

Failed to run `config` for indent-blankline.nvim
                                                                                                                                                                                             
.../share/nvim/lazy/indent-blankline.nvim/lua/ibl/utils.lua:22: type: expected function: 0x7794a601d0d8, got string (HIGHLIGHT_SETUP)                                                        
                                                                                                                                                                                             
```
# stacktrace:                                                                                                                                                                                
  - vim/shared.lua:0 _in_ **validate**                                                                                                                                                       
  - /indent-blankline.nvim/lua/ibl/utils.lua:22 _in_ **validate**                                                                                                                            
  - /indent-blankline.nvim/lua/ibl/hooks.lua:61 _in_ **register**                                                                                                                            
  - init.lua:47 _in_ **config**                                                                                                                                                              
  - /NvChad/lua/nvchad/autocmds.lua:15  
```
and my config:

      local hooks = require "ibl.hooks"
      hooks.register(hooks.type.HIGHLIGHT_SETUP, function()
          vim.api.nvim_set_hl(0, "RainbowRed", { fg = "#E06C75" })
          vim.api.nvim_set_hl(0, "RainbowYellow", { fg = "#E5C07B" })
          vim.api.nvim_set_hl(0, "RainbowBlue", { fg = "#61AFEF" })
          vim.api.nvim_set_hl(0, "RainbowOrange", { fg = "#D19A66" })
          vim.api.nvim_set_hl(0, "RainbowGreen", { fg = "#98C379" })
          vim.api.nvim_set_hl(0, "RainbowViolet", { fg = "#C678DD" })
          vim.api.nvim_set_hl(0, "RainbowCyan", { fg = "#56B6C2" })
      end)



### Steps to reproduce


      local hooks = require "ibl.hooks"
      hooks.register(hooks.type.HIGHLIGHT_SETUP, function()
          vim.api.nvim_set_hl(0, "RainbowRed", { fg = "#E06C75" })
          vim.api.nvim_set_hl(0, "RainbowYellow", { fg = "#E5C07B" })
          vim.api.nvim_set_hl(0, "RainbowBlue", { fg = "#61AFEF" })
          vim.api.nvim_set_hl(0, "RainbowOrange", { fg = "#D19A66" })
          vim.api.nvim_set_hl(0, "RainbowGreen", { fg = "#98C379" })
          vim.api.nvim_set_hl(0, "RainbowViolet", { fg = "#C678DD" })
          vim.api.nvim_set_hl(0, "RainbowCyan", { fg = "#56B6C2" })
      end)

### Expected behavior

not error

### Neovim version (nvim -v)

NVIM v0.11.0-dev Build type: RelWithDebInfo LuaJIT 2.1.1703358377 Run "nvim -V1 -v" for more info

**Comment by HugoPhi** [comment]
I was version v0.11-dev and the bug is solved when I use v0.10 now.

**Comment by lukas-reineke** [comment]
v0.11 works as well, you just need to use the latest one.

v0.11 is still unstable. It changes every day. It is not possible to support every iteration. Only the latest one is supported.
